nav2_costmap_2d/plugins/costmap_filters/keepout_filter.cpp

```diff
@@ -35,12 +35,13 @@
  * Author: Alexey Merzlyakov
  *********************************************************************/
 
-#include "nav2_costmap_2d/costmap_filters/keepout_filter.hpp"
-
 #include "tf2/convert.h"
```

Comment:
There's still a bunch of keepout filter commits from our last PR, please get rid of them.

Reply:
OK. Got it. Moving KeepoutFilter related clean-ups to separate PR #2101.